sdv/multi_table/base.py

```diff
@@ -38,8 +40,8 @@ class BaseMultiTableSynthesizer:
     multi table synthesizers need to implement, as well as common functionality.
 
     Args:
-        metadata (sdv.metadata.multi_table.MultiTableMetadata):
-            Multi table metadata representing the data tables that this synthesizer will be used
+        metadata (sdv.metadata.Metadata):
+            Table metadata representing the data tables that this synthesizer will be used
```

Comment:
Maybe rephrase to just "Metadata representing the..."?

Reply:
Fixed